app/views/categories/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= render "header" %>
```

Comment:
Please fix the following issues that are causing the UI to not show properly 😥 As the project's target is mobile, all the UI elements must fit for the mentioned sizes 

![image](https://user-images.githubusercontent.com/15807343/195507759-dc5a61bb-e872-46ea-8abb-470003e55b45.png)